venv/bin/Activate.ps1

```diff
@@ -0,0 +1,241 @@
+<#
+.Synopsis

```

Comment:
It is convention to add the `venv/` directory to the `.gitignore`. I don't think we need to commit all of these files. :sweat_smile:

Reply:
The entry of `venv/` was there on `.gitignore`. :laughing: I kinda added them even when the `requirements_venv,txt` could have just done fine. I'll remove them.